src/main/java/io/vertx/ext/eventbus/bridge/tcp/impl/TcpEventBusBridgeImpl.java

```diff
@@ -124,7 +125,8 @@ public TcpEventBusBridge listen(int port, Handler<AsyncResult<TcpEventBusBridge>
 
   private void handler(NetSocket socket) {
 
-    final Map<String, MessageConsumer<?>> registry = new HashMap<>();
+    final Map<String, MessageConsumer<?>> registry = new ConcurrentHashMap<>();
+    final Map<String, Message<JsonObject>> replies = new ConcurrentHashMap<>();
```

Comment:
I'm not a Vert.x expert, but i have a concern : in the case where we have multiple instances, say instance A sends the message through the NetServer and stores the message in the `replies` map, the NetClient that receives this message will eventually send a new message to the NetServer, this message can be processed by instance B, which will have a different `replies` map.
Like I said, i'm no expert, I may be completely wrong, and if I am could you enlighten my concerns.